demo/README.md

```diff
@@ -0,0 +1,60 @@
+# Hateless demo
+
+The api let its users to analyze the content of a text message. Result returned consists of the api's classification label for the text. There are three labels:
+
+- Label '0' - `Hate speech`
+- Label '1' - `Offensive speech`
+- Label '2' - `Neither`
+
+## Changelogs
+
+This section keeps track of changes in this documentation
+
+| Date           | Author       | Description    |
+| -------------- | ------------ | -------------- |
+| June 6th, 2022 | Nguyen (Jim) | README created |
+
+A demo for Hateless webservice—see [Online DEMO](https://hateless.herokuapp.com)
+
+![Demo_Thumb](./images/demo_thumb.png)
+
+## Specifications
+
+### 0. Overview
+
+```txt
+./demo
+|---- README       // Project structures, specs, etc.
+|---- web_api      // Backend sub-project
+|---- web_app      // Frontend sub-project
+|---- docs         // Engineering documentations
+```
+
+Two main components for project:
+
+- Backend API
+- Frontend web application
+
+**Note:**
+
+- Frontend web app is built and delivered by backend api at root
+- To run the demo on your local, please re-create virtual environment with dependency listed in `web_api/requirements.txt` then execute the `wsgi.py` with the venv's Python
+- To run the frontend only, change directory to `web_app/hateless` then run `npm run start`

```

Comment:
Which npm dependencies are required to run this? It would help me to have a short "Getting Started" section which briefly describes the steps i need to do so that i can run this application.

Reply:
The npm dependencies can be found in [package.json file](https://github.com/onurem/GSE---TEAM-A/blob/feat/api/demo/web_app/hateless/package.json)

Basically just React and MUI (for material UI components)

Reply:
I'll add the `Getting Started` section tonight